packages/cli/src/commands/account/get-attestations.ts

```diff
@@ -0,0 +1,99 @@
+import { ContractKit } from '@celo/contractkit'
+import { OdisUtils } from '@celo/identity'
+import { AuthSigner } from '@celo/identity/lib/odis/query'
+import { flags } from '@oclif/command'
+import Web3 from 'web3'
+import { BaseCommand } from '../../base'
+
+export default class GetAttestations extends BaseCommand {
+  static description =
+    "Looks up attestations associated with the provided phone number. If a pepper is not provided, it uses the --from account's balance to query the pepper."
+
+  static flags = {
+    ...BaseCommand.flags,
+    phoneNumber: flags.string({
+      required: false,
+      description: 'Phone number to check attestations for',
+    }),
+    from: flags.string({
+      required: false,
+      description: 'Account whose balance to use for querying ODIS for the pepper lookup',
+    }),
+    pepper: flags.string({
+      required: false,
+      description: 'ODIS phone number pepper',
+    }),
+    identifier: flags.string({
+      required: false,
+      description: 'On-chain identifier',
+    }),
+    network: flags.string({
+      required: false,
+      description: 'The ODIS service to hit: mainnet, alfajores, alfajoresstaging',
+    }),
+  }
+
+  static examples = [
+    'get-attestations --phoneNumber +15555555555 --from 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95',
+    'get-attestations --phoneNumber +15555555555 --pepper XgnKVpplZc0p1',
+    'get-attestations --identifier 0x4952c9db9c283a62721b13f56c4b5e84a438e2569af3de21cb3440efa8840872',
+  ]
+
+  async run() {
+    const res = this.parse(GetAttestations)
+    const phoneNumber = res.flags.phoneNumber
+    const account = res.flags.from
+    let identifier = res.flags.identifier
+    let pepper = res.flags.pepper
+    if (!account && !pepper && !identifier) {
+      console.error('Must specify either --from or --pepper or --identifier')
+      return
+    }
+    const network = res.flags.network
+
+    const web3 = new Web3()
+    const attestations = await this.kit.contracts.getAttestations()
+
+    if (!identifier) {
+      if (!phoneNumber) {
+        console.error('Must specify phoneNumber if identifier not provided')
+      }
+      // Get Phone number pepper
+      // Needs a balance to perform query
+      if (!pepper) {
+        pepper = await this.getPhoneNumberPepper(this.kit, phoneNumber!, account!, network)
+        console.log('Pepper: ' + pepper)
+      }
+
+      const computedIdentifier = web3.utils.soliditySha3({
```

Comment:
nit: would suggest using the kit for this, just to standardize how we call web3 functions
Then, you can also remove the web3 variable on line 54. 

```suggestion
      const computedIdentifier = this.kit.connection.web3.utils.soliditySha3({
```